pkgs/games/gale/default.nix

```
@@ -0,0 +1,82 @@
+{ lib
+, fetchFromGitHub
+, buildNpmPackage
+, openssl
+, pkg-config
+, freetype
+, libsoup
+, gtk3
+, webkitgtk
+, nodejs-slim
+, cargo-tauri
+, cargo
+, rustPlatform
+, rustc
+}:
+
+buildNpmPackage rec {
+
+  pname = "gale";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner =  "Kesomannen";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-yo/Mcvrt/Kn1Zlz7lbL3n7eyT2L9dJA5PTGgWLuxJ9M=";
+  };
```

```suggestion
buildNpmPackage (finalAttrs: {

  pname = "gale";
  version = "0.4.0";

  src = fetchFromGitHub {
    owner = "Kesomannen";
    repo = "${finalAttrs.pname}";
    rev = "${finalAttrs.version}";
    sha256 = "sha256-yo/Mcvrt/Kn1Zlz7lbL3n7eyT2L9dJA5PTGgWLuxJ9M=";
  };
```

`rec` is apparently old-hat and `finalAttrs` is the future. [The manual shows otherwise for buildNpmPackage](https://nixos.org/manual/nixpkgs/unstable/#javascript-tool-specific), but this is a suggestion I always see in PRs.